MaxMind-DB-spec.md

```diff
@@ -221,7 +221,7 @@ The reason that we subtract the `$node_count` is best demonstrated by an example
 Let's assume we have a 24-bit tree with 1,000 nodes. Each node contains 48
 bits, or 6 bytes. The size of the tree is 6,000 bytes.
 
-When a record in the tree contains a number that is < 1,000, this is a *node
+When a record in the tree contains a number that is &lt; 1,000, this is a *node
```

Comment:
Given that this document is intended to be human readable as a text file, we might be better off changing this to "less than".